chess-AI/mcts.py

```diff
@@ -1,8 +1,11 @@
 from math import sqrt
-
+# import pandas as pd
 import numpy as np
-
 from state_representation import get_cnn_input
+from output_representation import PlayNetworkPolicyConverter
+import torch
+import random
```

Comment:
Similar to above comment, [style guide](https://google.github.io/styleguide/pyguide.html#313-imports-formatting) says: "Within each grouping, imports should be sorted lexicographically, ignoring case, according to each module’s full package path (the path in from path import ...). Code may optionally place a blank line between import sections."